Symfony/CS/Tests/AbstractIntegrationTest.php

```diff
@@ -78,15 +80,15 @@ public static function tearDownAfterClass()
      *
      * @see doTestIntegration()
      */
-    public function testIntegration($testFileName, $testTitle, $fixers, array $requirements, $expected, $input = null)
+    public function testIntegration(IntegrationSuite $suite)
     {
-        $this->doTestIntegration($testFileName, $testTitle, $fixers, $requirements, $expected, $input);
+        $this->doTestIntegration($suite);
     }
 
     /**
      * Creates test data by parsing '.test' files.
      *
-     * @return array
+     * @return IntegrationSuite[]
```

Comment:
That would be the prefered syntax that's actually compatible with php-cs-fixer atm.